webpack/scenes/ContentViews/Details/Filters/ContentType.js

```diff
@@ -0,0 +1,40 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import RepoIcon from '../Repositories/RepoIcon';
+import { capitalize } from '../../../../utils/helpers';
+
+const typeName = (type, errataByDate) => {
+  if (errataByDate) return 'Errata - by date range';
+  const nameMap = {
+    rpm: __('RPM'),
+    docker: __('Container image tag'),
+    modulemd: __('Module stream'),
+    erratum: __('Errata'),
+  };
+
+  if (Object.prototype.hasOwnProperty.call(nameMap, type)) return nameMap[type];
+  return capitalize(type.replace('_', ' '));
+};
+
+const ContentType = ({ type, errataByDate }) => {
+  const repoType = ['rpm', 'modulemd', 'rpm', 'erratum', 'package_group'].includes(type) ? 'yum' : type;
+  return (
+    <Fragment>
```

Comment:
We should really figure out what we need to do to make our Babel recognize `<> </>`...

Reply:
Apparently this was added in Babel 7.0.0: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#babel

Since we're using v7.7 in our `vendor` `devDependencies`, I would think maybe it should work now?